api/src/main/java/uk/gov/dhsc/htbhf/dwp/entity/Household.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.dhsc.htbhf.dwp.entity;
+
+import java.util.Set;
+
+public interface Household {
```

Comment:
Can we have some JavaDoc on this interface please?

Reply:
I'm not sure what value that would add besides just stating the method signatures.